CHANGELOG.unreleased.md

```diff
@@ -11,6 +11,7 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [Perf Alert] Fix storage usage alerts triggered by particular SRs (PR [#6755](https://github.com/vatesfr/xen-orchestra/pull/6755))
```

Comment:
1. always add new enhancement/bug fix entries at the end
2. this is too vague, *particular* does not mean anything

Message proposal:

```suggestion
- [Plugin/perf-alert] Ignore special SRs (e.g. *XCP-ng Tools*, *DVD drives*, etc) as their usage is always 100% (PR [#6755](https://github.com/vatesfr/xen-orchestra/pull/6755))
```